features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/admin/site/themes/wso2/templates/policy/resource/policy-add/js/policy-designer-new.js

```diff
@@ -265,13 +265,20 @@ var validIPRange = function (startIP, endIP) {
     }
     var startIPBlocks = startIP.split(".");
     var endIPBlocks = endIP.split(".");
-
+    var startIp = 0;
+    var endIp = 0;
     for (var i = 0; i < 4; i++) {
-        if (startIPBlocks[i] >= endIPBlocks[i]) {
-            return false;
-        }
+        var power = 3-i;
```

Comment:
Cant we initiate **var power** outside loop?

Reply:
fixed

Reply:
fixed in 61f9d95, b02cc84